workflow/environments/README.md

```diff
@@ -20,7 +20,7 @@ Requirements: Setup github SSH keys for maui,
 
 A user specific environment of workflow, qcore, IMCalc, Empirical Engine and Pre-processing 
 can be created as follows:  
-1) Go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy
+1) Log in to Maui, and go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy. If you don't have it, `git clone` now.
 2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi"
```

Comment:
This `...` is relative, but you don't say relative to where. In my case I:

1. Cloned `slurm_gm_workflow` in my home directory
2. Navigated to `slurm_gm_workflow/workflow/environments/org/nesi`

I think this should not have a relative path. Maybe explicitly state to work from `$HOME` and then navigate to 
`~/slurm_gm_workflow/workflow/environments/org/nesi`

Reply:
I agree that changing this to include the full path might be better to avoid potential confusion